datalad_container/containers_add.py

```diff
@@ -57,6 +57,8 @@ def _guess_call_fmt(ds, name, url):
         return 'singularity exec {img} {cmd}'
     elif url.startswith('dhub://'):
         return op.basename(sys.executable) + ' -m datalad_container.adapters.docker run {img} {cmd}'
+    elif url.startswith('ecr://'):
+        return op.basename(sys.executable) + ' -m datalad_container.adapters.docker run {img} {cmd}'
```

Comment:
But if there is no difference (special handling for ecr), we could just still just use them with dhub://, don't we?

Reply:
actually ya.. I should've tried that first! it actually works fine with `dhub://`... It is a little unclear though since it is another registry, and it requires a few steps to login first.